client/my-sites/domains/domain-management/transfer/transfer-out/icann-verification.jsx

```diff
@@ -13,52 +14,52 @@ import Button from 'components/button';
 import notices from 'notices';
 import support from 'lib/url/support';
 
-const IcannVerification = React.createClass( {
-	getInitialState() {
-		return {
-			submitting: false
-		};
-	},
+class IcannVerification extends React.Component {
```

Comment:
Maybe we could re-use the new ICANN verification notice from domain management here?